src/shared_modules/indexer_connector/src/indexerConnector.cpp

```diff
@@ -414,7 +414,7 @@ IndexerConnector::IndexerConnector(
                     [](const std::string& response) { logDebug2(IC_NAME, "Response: %s", response.c_str()); },
                     [](const std::string& error, const long statusCode)
                     {
-                        logError(IC_NAME, "%s, status code: %ld.", error.c_str(), statusCode);
+                        logError(IC_NAME, "%s, status code: %ld", error.c_str(), statusCode);
```

Comment:
Restore the missing dot
```suggestion
                        logError(IC_NAME, "%s, status code: %ld.", error.c_str(), statusCode);
```

Reply:
You are right, thank you!